project/Build.scala

```diff
@@ -44,20 +46,22 @@ object ApplicationBuild extends Build {
     val appName = "metrics-portal"
     val akkaVersion = "2.4.18"
     val akkaHttpVersion = "10.0.6"
-    val jacksonVersion = "2.7.4"
+    val jacksonVersion = "2.9.0"
```

Comment:
2.9.2? -- I'll be pushing our other libs to this.

Reply:
Updated.